src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -481,6 +485,8 @@ private void QueryResults()
                             {
                                 if (!plugin.Metadata.Disabled)
                                 {
+                                    Query query;
+                                    pluginQueryPair.TryGetValue(plugin, out query);
```

Comment:
[nit] This is not required as `TryGetValue` will always be true here since we are iterating on the keys. We can instead do `foreach(KeyValuePair<PluginPair, Query> entry in myDictionary)` to access plugin pair and query.